src/org/javarosa/core/io/Std.java

```diff
@@ -1,32 +0,0 @@
-package org.javarosa.core.io;
-
-import java.io.PrintStream;
-
-/**
- * Used to break dependency directly on System.out and System.err.
- */
-public class Std {
```

Comment:
Did you coordinate with @batkinson, who created this in July?
https://github.com/dcbriccetti/javarosa/commit/f97c27dcf0fbcb39775bde54cd2b4143d5954765

Reply:
Not really... It was very useful to guide the refactors, though...

Reply:
This will be a breaking change for him. @lognaturel, should we ask for Brent’s review of this change?

Reply:
@batkinson wasn't on the TSC call today so I'm guessing he has not had Internet access. :/

How about leaving the `Std` class in, marking it as deprecated and just not using it internally anymore? We could do this with other public classes removed. If we're removing public interfaces we probably should do a major release. By leaving them in we are sure clients can upgrade without changes (even though they may not get the expected logging anymore).

Reply:
Sounds good to me.

Reply:
OK, I'll review this option and get back with feedback... Let's hold this PR in the meantime